recipe.py

```diff
@@ -45,6 +46,8 @@ def set_ingredients(self,ingredients):
     def remove_ingredient(self,ingredient):
         self.ingredients.remove(ingredient)
 
+    def display_ingredients(self):
```

Comment:
Its possible that whatever calls this will want to format or handle the display in a different way to its possibly better that this reports back a formatted string (and if a separate ingredients class is created this will pass back formatted strings).

Reply:
Okay I will make it so it returns the ingredients in a nice format